src/main/java/uk/gov/hmcts/reform/cpo/service/mapper/CasePaymentOrderMapper.java

```diff
@@ -15,12 +15,14 @@
 
     @Mapping(target = "id", ignore = true)
     @Mapping(target = "createdTimestamp", ignore = true)
+    @Mapping(target = "historyExists", expression = "java(false)")
     CasePaymentOrderEntity toEntity(CreateCasePaymentOrderRequest createCasePaymentOrderRequest,
                                                 String createdBy);
 
     CasePaymentOrder toDomainModel(CasePaymentOrderEntity casePaymentOrderEntity);
 
     @Mapping(target = "createdTimestamp", expression = "java(java.time.LocalDateTime.now())")
+    @Mapping(target = "historyExists", expression = "java(true)")
```

Comment:
we can make things more "explicit" and obvious if we had the Service setting this property to true on the entity in `updateCasePaymentOrder` method, rather than the Mapper.

Reply:
It took me ages to work out where this was being done, so second that

Reply:
probably we should do same for "createdTimestamp" as well @mario-paniccia . If so do you mind to clean-up in this PR @KleoG ?